lib/utils/getoptions.js

```diff
@@ -38,10 +40,19 @@ module.exports = function cwdResolver( callOptions, cwd ) {
 
 	options.packages = path.resolve( cwd, options.packages );
 
+	/* istanbul ignore if */
```

Comment:
?

Reply:
Removed the comment and added a test for such a scenario.